instagram-app/src/App.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import './App.css';
+import dummyData from './dummy-data'
+import SearchBar from './components/SearchBar/SearchBar';
+import CommentSection from './components/CommentSection/CommentSection';
+import PostContainer from './components/PostContainer/PostContainer';
+import '../src/components/PostContainer/PostContainer.css'
+import PostsPage from './components/PostContainer/PostsPage'
+import withAuthenticate from '../src/components/authentication/withAuthenticate'
+import Login from './components/Login/LoginPage'
+
+
+const ComponentFromWithAuthenticate = withAuthenticate(PostsPage)(Login)
+
+class App extends React.Component {
+  state = {
+    stateData: dummyData,
+  }
+
+  login = (userName, password) => {
+    localStorage.setItem('username', userName);
+  }
+
+  logout = () => {
+    localStorage.setItem('username', '')
```

Comment:
I know we talked about this last night but it might be better to remove username from localStorage as opposed to saving a blank value.